utils/size.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"fmt"
+)
+
+type DataSize float64
+
+//nolint:mnd
+func (d DataSize) String() string {
+	switch {
+	case d >= 1099511627776:
+		return fmt.Sprintf("%.2f TiB", d/1099511627776)
+	case d >= 1073741824:
+		return fmt.Sprintf("%.2f GiB", d/1073741824)
+	case d >= 1048576:
```

Comment:
let's replace it with constants from const.go (there is only 1 for Megabyte, but you can add the rest)